src/main/java/edu/kis/powp/jobs2d/drivers/DriverChangeEventSubscriber.java

```diff
@@ -3,7 +3,11 @@
 import edu.kis.powp.jobs2d.features.DriverFeature;
 import edu.kis.powp.observer.Subscriber;
 
-public class SelectEventSubscriber implements Subscriber {
+/**
+ * This class subscribes to the DriverManager's changes (invoked by setCurrentDriver())
+ */
+
+public class DriverChangeEventSubscriber implements Subscriber {
```

Comment:
I agree that this subscriber waits for "driver change" but the point of all driver manager subscribers would be to wait for "driver change". The thing that make them differ is question why they are waiting (what they are going to do "about it") This should be at least mentioned (a hint).